__tests__/lib/dev-environment/dev-environment-core.js

```diff
@@ -25,22 +25,28 @@ import { getEnvironmentPath,
 import { searchAndReplace } from '../../../src/lib/search-and-replace';
 import { resolvePath } from '../../../src/lib/dev-environment/dev-environment-cli';
 import { DEV_ENVIRONMENT_NOT_FOUND } from '../../../src/lib/constants/dev-environment';
+import { bootstrapLando } from '../../../src/lib/dev-environment/dev-environment-lando';
 
 jest.mock( 'xdg-basedir', () => ( {} ) );
-jest.mock( 'fs' );
```

Comment:
Now that we call `bootstrapLando()` before calling our Lando wrappers, we cannot safely mock the `fs` module because Lando needs it.

Instead, we are going to mock only those functions we are interested in.